vipps-order-management-api-quick-start.md

```diff
@@ -3,116 +3,368 @@
 title: Quick start for the Order Management API
 sidebar_label: Quick start
 sidebar_position: 10
-description: Quick start guide for the using the Order Management API with Postman.
+description: Quick steps for getting started with the Order Management API.
+toc_min_heading_level: 2
+toc_max_heading_level: 5
 pagination_next: null
 pagination_prev: null
 ---
+
+import ApiSchema from '@theme/ApiSchema';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
 END_METADATA -->
 
 # Quick start
 
 Use the Order Management API to generate enriched receipts.
 
-<!-- START_COMMENT -->
-
-ℹ️ Please use the new documentation:
-[Vipps MobilePay Technical Documentation](https://developer.vippsmobilepay.com/docs/APIs/order-management-api).
+## Before you begin
 
-## Table of Contents
+This document covers the quick steps for getting started with the Order Management API.
+You must have already signed up as an organization with Vipps MobilePay and have
+your test credentials from the merchant portal, as described in the
+[Getting started guide](https://developer.vippsmobilepay.com/docs/getting-started).
 
-* [Quick start](#quick-start)
-  * [Table of Contents](#table-of-contents)
-  * [Postman](#postman)
-    * [Prerequisites](#prerequisites)
-    * [Step 1: Get the Postman collection and environment](#step-1-get-the-postman-collection-and-environment)
-    * [Step 2: Import the Postman files](#step-2-import-the-postman-files)
-    * [Step 3: Set up Postman environment](#step-3-set-up-postman-environment)
-  * [Make API calls](#make-api-calls)
+**Important:** The examples use standard example values that you must change to
+use *your* values. This includes API keys, HTTP headers, reference, etc.
 
-<!-- END_COMMENT -->
+## Your first receipt
 
-## Postman
+### Step 1 - Setup
 
-### Prerequisites
+<Tabs
```

Comment:
![image](https://github.com/vippsas/vipps-order-management-api/assets/117826285/5e8cea37-16d1-4980-94cf-d41e29fd4a9c)
 is it supposed to be like this?

Reply:
@FlameboyVippsMobilePay Do you mean with the code <Tabs ..... ? Yes, it looks nice in Docusaurus.

Here's an example: https://developer.vippsmobilepay.com/docs/APIs/epayment-api/quick-start/

We decided a while back that we could include some nice features even though they don't look as nice if the user is viewing the page with GitHub. We try to funnel all the users to the website.

Reply:
aahhh got it